react-app/src/view/Categories/ListOfCategories.jsx

```diff
@@ -0,0 +1,85 @@
+import { Button, Typography } from "@mui/material";
+import { Container } from "@mui/system";
+import React, { useState, useEffect } from "react";
+// import axiosInstance from "../../axios";
+// import { base } from "../../api";
+import Table from "../../components/Table";
+import { useNavigate } from "react-router-dom";
+import PopupCreate from "./PopupCreate";
+import { getCategories } from "../../service";
+
+const ListOfCategories = () => {
+  const [listOfCategories, setListOfCategories] = useState([]);
+  const [openPopup, setOpenPopup] = useState(false);
+
+  const [cols] = useState([
+    {
+      name: "id",
+      label: "ID",
+    },
+    {
+      name: "name",
+      label: "Name",
+    },
+    {
+      action: "action",
+      label: "Action",
+    },
+  ]);
+
+  const navigate = useNavigate();
+
+  // const getCategories = async () => {
+  //   try {
+  //     const { data } = await axiosInstance({
+  //       method: "get",
+  //       url: base + "/categories",
+  //     });
+  //     console.log(data?.data);
+  //   } catch (errro) {
+  //     console.log("not successful");
+  //   }
+  // };
+
+  const test = async () => {
+    const data = await getCategories();
+    console.log(data.data);
+    setListOfCategories(data?.data);
+  };
```

Comment:
Name your functions more properly `getCategories` instead of `test`

Reply:
done!